cmd/skopeo/layers.go

```diff
@@ -45,6 +44,12 @@ func (opts *layersOptions) run(args []string, stdout io.Writer) (retErr error) {
 		return errors.New("Usage: layers imageReference [layer...]")
 	}
 
+	imageNames := make([]string, 1)
+	imageNames[0] = args[0]
+	if err := needReexec(imageNames); err != nil {
```

Comment:
This would be easier to read/write with a
```go
func needReexec(imageNames ...string)
```
; no need to have an array, or artificially manufacture it, in the callers.

Reply:
Thanks for the suggestion (i didn't know golang had varargs... :).